mitodl/mit-learn
# restore images in subscription emails

### Description (What does it do?)
This PR:
- restores an image file that was statically referenced in subscription emails 
- Adds a default fallback image for resources in the email that dont have images


### How can this be tested?
- Follow instructions on [this PR](https://github.com/mitodl/mit-learn/pull/1619) to send yourself a subscription email (email must be opened on the same machine your server is running on to see images render)
